### DISCUSSION TITLE
is this course really Good?

### DISCUSSION THREAD

**Discussion by abhisheksd27** [Cyfrin/foundry-full-course-cu#2186] [General]
i want to study and learn blockchain ...  so is this the correct place to start ?

**Comment by chapman91** [comment]
Very good! Been going at it for a few weeks now.

**Comment by EngrPips** [comment]
> i want to study and learn blockchain ... so is this the correct place to start ?

This is the best place you can be, buddy! Welcome aboard and i wish you the best on your journey.
